fairlearn/datasets/_fetch_acs_income.py

```diff
@@ -86,12 +86,17 @@ def fetch_acs_income(
             Array of ordered feature names used in the dataset.
         DESCR : string
             Description of the ACSIncome dataset.
-
-    (data, target) : tuple of (numpy.ndarray, numpy.ndarray)
-        if ``return_X_y`` is True and ``as_frame`` is False
-
-    (data, target) : tuple of (pandas.DataFrame, pandas.Series)
-        if ``return_X_y`` is True and ``as_frame`` is True
+        categories : dict or None
+            Maps each categorical feature name to a list of values, such that the
+            value encoded as i is ith in the list. If ``as_frame`` is True, this is None.
+        frame : pandas DataFrame
+            Only present when ``as_frame`` is True. DataFrame with ``data`` and ``target``.
+        
+    (data, target) : tuple if ``return_X_y`` is True
+
+    Notes
```

Comment:
I don't think that we need this note. But, if you want to acknowledge the relationship with fetch_openml, we could say that our API largely follows the API of `sklearn.datasts.fetch_openml`.

Reply:
Sure. I can revise the comments in the note. I will send the updated pull. Let me know if that sounds ok to move forward:)